scripts/mitochondria_m2_wdl/ExampleInputsMitochondriaPipeline.json

```diff
@@ -1,8 +1,11 @@
 {
   "MitochondriaPipeline.wgs_aligned_input_bam_or_cram": "input_bam_here",
   "MitochondriaPipeline.wgs_aligned_input_bam_or_cram_index": "input_bam_index_here",
-  "MitochondriaPipeline.autosomal_coverage": autosomal_median_coverage_here,
+  "MitochondriaPipeline.autosomal_coverage": "autosomal_median_coverage_here",
```

Comment:
I think we left the quotes off in this example to indicate that this was an Int instead of a String. Do you think it would be clearer to put an actual example here like 30?